### ISSUE TITLE
Missing deploy option in configurations

### ISSUE THREAD

**Issue by thisbarker** [XamarinUniversity/AND205#2]
https://github.com/XamarinUniversity/AND205/blob/33bddd6c18a0ac6cd76a6f8a8db1e3d968fd3aef/Exercise%205/Start/Clock/Clock.sln#L14

```
{D55C3194-FA80-434A-875B-E31A2C70ACFF}.Debug|Any CPU.Deploy.0 = Debug|Any CPU
{D55C3194-FA80-434A-875B-E31A2C70ACFF}.Release|Any CPU.Deploy.0 = Release|Any CPU
```

**Comment by patridge** [comment]
Thank you for filing this. I was able to reproduce this and will fix it soon.

**Comment by patridge** [comment]
Fixed in 1b80bc3ae6059282546b7acf6a18a0e0f999759c
